### ISSUE TITLE
Compilation Error

### ISSUE THREAD

**Issue by sameerkhurana10** [dkpro/similarity-gpl-tmp#1]
Hi, 

Getting a compilation error when doing 'mvn clean install -DskipTests'.

[INFO] dkpro.similarity.experiments.sts-2013-baseline-gpl . FAILURE [  1.280 s]
[INFO] dkpro.similarity.experiments.rte-gpl ............... SKIPPED
[INFO] dkpro.similarity.experiments.sts-2013-gpl .......... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 25.472 s
[INFO] Finished at: 2015-07-13T06:58:21+01:00
[INFO] Final Memory: 49M/264M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project dkpro.similarity.experiments.sts-2013-baseline-gpl: Compilation failure: Compilation failure:
[ERROR] /Users/sameerkhurana10/git/similarity-gpl-tmp/dkpro.similarity.experiments.sts-2013-baseline-gpl/src/main/java/dkpro/similarity/experiments/sts2013baseline/util/Evaluator.java:[33,48] package org.apache.commons.math.stat.correlation does not exist
[ERROR] /Users/sameerkhurana10/git/similarity-gpl-tmp/dkpro.similarity.experiments.sts-2013-baseline-gpl/src/main/java/dkpro/similarity/experiments/sts2013baseline/util/Evaluator.java:[273,25] cannot find symbol
[ERROR] symbol:   class PearsonsCorrelation
[ERROR] location: class dkpro.similarity.experiments.sts2013baseline.util.Evaluator
[ERROR] /Users/sameerkhurana10/git/similarity-gpl-tmp/dkpro.similarity.experiments.sts-2013-baseline-gpl/src/main/java/dkpro/similarity/experiments/sts2013baseline/util/Evaluator.java:[273,59] cannot find symbol
[ERROR] symbol:   class PearsonsCorrelation
[ERROR] location: class dkpro.similarity.experiments.sts2013baseline.util.Evaluator
[ERROR] /Users/sameerkhurana10/git/similarity-gpl-tmp/dkpro.similarity.experiments.sts-2013-baseline-gpl/src/main/java/dkpro/similarity/experiments/sts2013baseline/util/Evaluator.java:[329,17] cannot find symbol
[ERROR] symbol:   class PearsonsCorrelation
[ERROR] location: class dkpro.similarity.experiments.sts2013baseline.util.Evaluator
[ERROR] /Users/sameerkhurana10/git/similarity-gpl-tmp/dkpro.similarity.experiments.sts-2013-baseline-gpl/src/main/java/dkpro/similarity/experiments/sts2013baseline/util/Evaluator.java:[329,51] cannot find symbol
[ERROR] symbol:   class PearsonsCorrelation
[ERROR] location: class dkpro.similarity.experiments.sts2013baseline.util.Evaluator
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :dkpro.similarity.experiments.sts-2013-baseline-gpl

**Comment by zesch** [comment]
Yes, you are right. We are in the process of moving the temporary gpl repository into a shared repository for all similarity modules regardless of license.
Thus, we haven't fixed this here.
Leaving the issue open as a reminder to prioritize the move a bit higher.
